packages/cozy-sharing/src/state.js

```diff
@@ -384,9 +384,16 @@ export const getSharingDocIds = sharing => {
   return docs
 }
 
-const getPermissionDocIds = perm =>
+// Some permissions can not have values since they can
+// be on a global doctype. In that case, we can't sort
+// them by id
+export const getPermissionDocIds = perm =>
   Object.keys(perm.attributes.permissions)
-    .map(k => perm.attributes.permissions[k].values)
+    .map(k =>
+      perm.attributes.permissions[k].values
+        ? perm.attributes.permissions[k].values
+        : []
+    )
     .reduce((acc, val) => [...acc, ...val], [])
```

Comment:
Nit: mapValues + flatten also does the job.

flatten(mapValues(perm.attributes.permissions, x => x || []))